notion/manifest.json

```diff
@@ -0,0 +1,42 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "bb61af26-0004-11ee-81a2-da7ad0900002",
+  "app_id": "notion",
+  "display_on_public_website": false,
```

Comment:
```suggestion
  "display_on_public_website": true,
```

Reply:
The integration is being released as a customer early access program for now, therefore, we cannot list publically.

Reply:
This field isn't related to the release to a specific customer account. Setting this to true will still allow us to enable the integration for a specific customer. Can you please update this to true? Thanks!